lib/modules/manager/github-actions/extract.ts

```diff
@@ -182,6 +184,14 @@ function extractSteps(
   deps: PackageDependency<Record<string, any>>[],
 ): void {
   for (const step of steps) {
+    if (step.uses) {
```

Comment:
Won't parsing work without this explicit check?

Reply:
yes, remove this still work. 

do you want to this check?

Reply:
Checks like these are why we use schemas, so I would prefer to remove it